### ISSUE TITLE
Milestone 1 - Catalogue changes

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/NorthernAustralia#73]
A list of catalogue changes following the addition of data, as per Milestone 1.

**Comment by steve9164** [comment]
- [ ] move Soil category to the very bottom **Does this mean move to the bottom of All Data, R&E and Ag&F?**

**Comment by steve9164** [comment]
See http://prototype-northernaustralia.terria.io/ (updated with all changes except soil)

**Comment by steve9164** [comment]
Note: I've temporarily added both "Register of Native Title Claims" and a "Register of Native Title Claims (WMS)" before we decide which to choose of these layers (and what to do with the RNT Bodies Corporate layer)

**Comment by AnaBelgun** [comment]
Yes, please. Move soil to the very bottom of each category, including All Data.

**Comment by AnaBelgun** [comment]
@steve9164 Hi. I added one small change
